TideApp/TideApp/Domain/Location/UserLocator.swift

```diff
@@ -16,6 +16,7 @@ final class UserLocator : ObservableObject {
 
   func start() {
     cancellable = CLLocationManager.publishLocation()
+      .filter { $0 != self.location}
```

Comment:
Shout if you think there's a better way 😄

Reply:
That's an elegant solution.